addons/xeh/fnc_postInit.sqf

```diff
@@ -19,6 +19,17 @@ Author:
 isNil {
     XEH_LOG("PostInit started. " + PFORMAT_9("MISSIONINIT",missionName,missionVersion,worldName,isMultiplayer,isServer,isDedicated,CBA_isHeadlessClient,hasInterface,didJIP));
 
+    addMissionEventHandler ["EntityCreated", {
```

Comment:
we would only need to add this if:
we have incompatible vehicles
and
xeh were added to those vehicles

Reply:
I added a filter to execute the code only on objects that are not compatible. 

I also added a foreach to initialize all the incompatible XEH objects that were either created earlier or are initially present on the map, and therefore are not detected by `EntityCreate`